bindings/python/malamute/_malamute_ctypes.py

```diff
@@ -7,7 +7,7 @@
 import os, sys
 from ctypes import *
 from ctypes.util import find_library
-import czmq
+import czmq.git
```

Comment:
It looked weird to me, however I have no clue about python bindings. The best i can do is to find and revert the change in zproject and send the pr again.